tests/test_lag_2.py

```diff
@@ -40,244 +35,227 @@ def common_setup_teardown(duthost, ptfhost, testbed, conn_graph_facts):
     if testbed_type not in support_testbed_types:
         pytest.skip("Not support given test bed type %s" % testbed_type)
 
-    yield duthost, ptfhost, vm_neighbors, mg_facts, lag_facts, fanout_neighbors
+    yield duthost, ptfhost, lag_facts
+
+class TestLag:
+    def __init__(self, duthost, ptfhost, nbrhosts, fanouthosts, conn_graph_facts):
```

Comment:
Looks like this __init__ method is causing the warning. Can you try rename this to something like setup() and call it after creating the instance?

Looks like this is a common thing for pytest.

Reply:
Pytest will detect testcases for class method too if the class name begin with 'Test'.
I renamed the class and pytest will not raising this warning anymore.